src/Tizen.Network.Stc/Tizen.Network.Stc/StcManager.cs

```diff
@@ -0,0 +1,95 @@
+{
+    /// <summary>
+    /// A class for managing the Stc handle.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public sealed class SafeStcHandle : SafeHandle
+    {
+        internal SafeStcHandle() : base(IntPtr.Zero, true)
+        {
+        }
+
+        /// <summary>
+        /// Checks the validity of the handle.
+        /// </summary>
+        /// <value>Represents the validity of the handle.</value>
+        public override bool IsInvalid
+        {
+            get
+            {
+                return this.handle == IntPtr.Zero;
+            }
+        }
+
+        /// <summary>
+        /// Release the Stc handle
+        /// </summary>
+        /// <returns>True if the handle is released successfully, otherwise false.</returns>
+        protected override bool ReleaseHandle()
+        {
+            Interop.Stc.Deinitialize(this.handle);
+            this.SetHandle(IntPtr.Zero);
+            return true;
+        }
+    }
+
+    /// <summary>
+    /// A class which is used to manage Smart Traffic control (Stc).<br/>
+    /// </summary>
+    public static class StcManager
+    {
+        /// <summary>
+        /// Gets the Stc safe handle.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <returns>The instance of the SafeStcHandle.</returns>
+        /// <privilege>http://tizen.org/privilege/network.get</privilege>
+        /// <exception cref="NotSupportedException">Thrown when the Stc is not supported.</exception>
+        /// <exception cref="UnauthorizedAccessException">Thrown when the permission is denied.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when the method failed due to an invalid operation.</exception>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        internal static SafeStcHandle GetStcHandle()
+        {
+            return StcManagerImpl.Instance.GetSafeHandle();
+        }
+
+        /// <summary>
+        /// Gets statistics information of applications that used network in between specified timestamps and matches the given StatisticsFilter, asynchronously.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <param name="filter"> The StatisticsFilter object.</param>
+        /// <returns>A list of the NetworkStatistics objects.</returns>
+        /// <privilege>http://tizen.org/privilege/network.get</privilege>
+        /// <exception cref="NotSupportedException">Thrown when the Stc is not supported.</exception>
+        /// <exception cref="UnauthorizedAccessException">Thrown when the permission is denied.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when the method failed due to an invalid operation.</exception>
+        /// <exception cref="ArgumentException">Thrown when the method is provided with invalid argument.</exception>
+        public static Task<IEnumerable<NetworkStatistics>> GetAllStatisticsAsync(StatisticsFilter filter)
```

Comment:
Now, we can remove `All` keyword

Reply:
AppId null is means `Any'?

Reply:
Renamed 'GetAllStatisticsAsync' to 'GetStatisticsAsync'.
Yes, Appid null means 'Any app'.

Reply:
As per today's discussion with Mr. Tak, we switched from 'GetStatisticsAsync' to 'GetAllStatisticsAsync'. But now from the code flow, it is found that 'GetAllStatisticsAsync' does not consider app_id in its internal logic. It does pass app_id to stc-manager. We will shortly fix this issue in Core Stc package.
That is, either
1) 'GetStatisticsAsync' should tell the end of callbacks.
2) or, 'GetAllStatisticsAsync' should consider app_id in its logic.

Reply:
IMO, 1 is correct way. 
2 is not correct because it includes `All` in method name